src/screens/UserPortal/Organizations/Organizations.tsx

```
@@ -136,205 +153,181 @@ export default function organizations(): JSX.Element {
     <>
       {hideDrawer ? (
         <Button
           className={styles.opendrawer}
-          onClick={(): void => {
-            setHideDrawer(!hideDrawer);
-          }}
+          onClick={() => setHideDrawer(!hideDrawer)}
           data-testid="openMenu"
         >
-          <i className="fa fa-angle-double-right" aria-hidden="true"></i>
+          <i className="fa fa-angle-double-right" aria-hidden="true" />
```

Biome (1.9.4)

[error] Disallow aria-hidden="true" from being set on focusable elements.

aria-hidden should not be set to true on focusable elements because this can lead to confusing behavior for screen reader users.
Unsafe fix: Remove the aria-hidden attribute from the element.


(lint/a11y/noAriaHiddenOnFocusable)